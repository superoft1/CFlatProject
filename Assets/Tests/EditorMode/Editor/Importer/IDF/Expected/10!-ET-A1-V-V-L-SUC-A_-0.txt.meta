fileFormatVersion: 2
guid: b568a320e3efeb348802db7d5acd6f82
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
