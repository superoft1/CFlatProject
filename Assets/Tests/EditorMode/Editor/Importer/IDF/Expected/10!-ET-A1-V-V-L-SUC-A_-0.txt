Tee, , (1.0, 0.0, 0.0), (-116.6, 31.5, 2.9), True
(-116.3, 31.5, 2.9)
(-116.9, 31.5, 2.9)
(-116.6, 31.5, 3.1)
FreeVertex: 2
Pipe, , (1.0, 0.0, 0.0), (-115.8, 31.5, 2.9), False
(-116.3, 31.5, 2.9)
(-115.2, 31.5, 2.9)
FreeVertex: 0
Elbow90, , (0.0, 0.0, 1.0), (-114.7, 31.5, 2.9), False
(-115.2, 31.5, 2.9)
(-114.7, 31.5, 2.4)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-114.7, 31.5, 2.2), False
(-114.7, 31.5, 2.4)
(-114.7, 31.5, 2.0)
FreeVertex: 0
WeldNeckFlange, , (0.0, 0.0, -1.0), (-114.7, 31.5, 1.9), False
(-114.7, 31.5, 1.9)
(-114.7, 31.5, 2.0)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-114.7, 31.5, 1.7), False
(-114.7, 31.5, 1.5)
(-114.7, 31.5, 1.9)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-114.7, 31.5, 1.5), False
(-114.7, 31.5, 1.5)
(-114.7, 31.5, 1.5)
FreeVertex: 0
WeldNeckFlange, , (0.0, 0.0, 1.0), (-114.7, 31.5, 1.4), False
(-114.7, 31.5, 1.5)
(-114.7, 31.5, 1.4)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-114.7, 31.5, 1.3), False
(-114.7, 31.5, 1.4)
(-114.7, 31.5, 1.2)
FreeVertex: 0
Elbow90, , (0.0, 1.0, 0.0), (-114.7, 31.5, 0.8), False
(-114.7, 31.5, 1.2)
(-114.7, 31.1, 0.8)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-114.7, 31.0, 0.8), False
(-114.7, 31.1, 0.8)
(-114.7, 30.9, 0.8)
FreeVertex: 0
StubInReinforcingWeld, , (1.0, 0.0, 0.0), (-114.7, 30.9, 0.8), False
(-114.7, 30.9, 0.8)
(-114.7, 30.9, 0.8)
(-114.7, 30.9, 0.8)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-114.7, 30.9, 0.9), False
(-114.7, 30.9, 0.8)
(-114.7, 30.9, 1.0)
FreeVertex: 0
WeldNeckFlange, , (0.0, 0.0, 1.0), (-114.7, 30.9, 1.1), False
(-114.7, 30.9, 1.1)
(-114.7, 30.9, 1.0)
FreeVertex: 0
GateValve, , (0.0, 0.0, 1.0), (-114.7, 30.9, 1.2), False
(-114.7, 30.9, 1.3)
(-114.7, 30.9, 1.1)
FreeVertex: 0
BlindFlange, , (0.0, 0.0, -1.0), (-114.7, 30.9, 1.3), False
(-114.7, 30.9, 1.3)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-114.7, 30.9, 0.8), False
(-114.7, 30.9, 0.8)
(-114.7, 30.9, 0.8)
(-114.7, 30.9, 0.6)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-114.7, 30.9, 0.5), False
(-114.7, 30.9, 0.6)
(-114.7, 30.9, 0.5)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-114.7, 30.9, 0.5), False
(-114.7, 30.9, 0.4)
(-114.7, 30.9, 0.5)
FreeVertex: 1
Pipe, , (0.0, -1.0, 0.0), (-114.7, 30.8, 0.8), False
(-114.7, 30.9, 0.8)
(-114.7, 30.7, 0.8)
FreeVertex: 0
Tee, , (0.0, -1.0, 0.0), (-114.7, 30.5, 0.8), True
(-114.7, 30.2, 0.8)
(-114.7, 30.7, 0.8)
(-114.5, 30.5, 0.6)
FreeVertex: 0
WeldNeckFlange, , (0.7, 0.0, -0.7), (-114.5, 30.5, 0.5), False
(-114.4, 30.5, 0.5)
(-114.5, 30.5, 0.6)
FreeVertex: 0
BlindFlange, , (-0.7, 0.0, 0.7), (-114.4, 30.5, 0.5), False
(-114.4, 30.5, 0.5)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-114.7, 30.1, 0.8), False
(-114.7, 30.2, 0.8)
(-114.7, 29.9, 0.8)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-114.7, 29.8, 0.8), False
(-114.7, 29.9, 0.8)
(-114.7, 29.7, 0.8)
FreeVertex: 0
EccentricPipingReducer, , (0.0, 1.0, 0.0), (-114.7, 29.6, 0.8), False
(-114.7, 29.7, 0.8)
(-114.7, 29.5, 0.8)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-114.7, 28.9, 0.8), False
(-114.7, 29.5, 0.8)
(-114.7, 28.4, 0.8)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-114.7, 28.3, 0.8), False
(-114.7, 28.3, 0.8)
(-114.7, 28.4, 0.8)
FreeVertex: 1
FreeVertexTotal: 4
